doc/tutorial/index.rst

```diff
@@ -161,28 +160,61 @@ us that the first packet indeed contains a Read Request:
-the expected content files:
+We now run Zeek on that trace to perform the batch conversion:
 
 .. code::
 
-    # zeek -r tftp_rrq.pcap udp-contents
-    # ls udp-contents.orig.*
-    udp-contents.orig.1367411051.972852.dat
-    udp-contents.orig.1367411052.077243.dat
-    udp-contents.orig.1367411052.086300.dat
-    udp-contents.orig.1367411052.088995.dat
-    udp-contents.orig.1367411052.091675.dat
-    [...]
+    # zeek -r tftp_rrq.pcap policy/frameworks/spicy/record-spicy-batch SpicyBatch::filename=tftp_rrq.dat
+    tracking [orig_h=192.168.0.253, orig_p=50618/udp, resp_h=192.168.0.10, resp_p=69/udp]
+    tracking [orig_h=192.168.0.10, orig_p=3445/udp, resp_h=192.168.0.253, resp_p=50618/udp]
+    recorded 2 sessions total
+    output in tftp_rrq.dat
+
+This leaves a new ``spicy-driver`` batch file in ``tftp_rrq.dat``.
+(If we had left off, the ``SpicyBatch::filename`` argument, the
+default output name is ``batch.dat``.)
 
-Per the timestamps included with the names, the first file is the one
-containing our Read Request. We can pass that into our Spicy parser:
+Now we can pass that batch file into ``spicy-driver``:
 
 .. spicy-output:: rrq.spicy 2
-    :exec: cat tutorial/examples/udp-contents.orig.1367411051.972852.dat | spicy-driver %INPUT
-    :show-as: cat udp-contents.orig.1367411051.972852.dat | spicy-driver tftp.spicy
+    :exec: spicy-driver -F tutorial/examples/tftp_rrq.dat -P 69/udp%orig=TFTP::ReadRequest %INPUT
+    :show-as: spicy-driver -F tftp_rrq.dat -P 69/udp%orig=TFTP::ReadRequest tftp.spicy
+
+The one additional piece here is that we need to tell ``spicy-driver``
+on which packets inside the batch file to deploy our parser (because,
+in principle, the batch could contain many different protocols
+distributed over independent connections). We achieve that through
+``-P 69/udp%orig=TFTP::ReadRequest``, which specifies that we want to
+use the ``TFTP::ReadRequest`` on all originator-side UDP packets for
+any connections on port ``69/udp``. See :ref:`spicy-driver
+documentation <spicy-driver-batch>` for more on that syntax.
+
+.. note::
+
+    .. versionadded:: 1.13 parser aliases
+
+    That option ``-P`` (aka ``--parser-alias``) is a feature added to
+    Spicy in version 1.13. An alternative to using that option---which
+    works with older Spicy version as well---is providing a
+    :ref:`%port <unit_meta_data>` property inside the
+    ```TFTP::ReadRequest`` unit; the two mechanisms are having the
+    same effect.
+
+Altogether, this gives us an easy way to test our TFTP parser with
+actual packet data, without needing to switch to full Zeek integration
+yet.
+
+The batch mode  of ``spicy-driver`` is generally worth keeping in mind
+while developing a new analyzer: Even if the eventual goal is a Zeek
+analyzer, it usually proves easier to stay just with ``spicy-driver``
+as long as possible, before switching to work on the Zeek-side glue
+layer later. The same observation applies to debugging: tracking down
```

Comment:
bit of word smithing for the part after the colon:

> even if the eventual goal is to create a Zeek analyzer, it is usually easier to work with spicy-driver for as long as possible before transitioning to the Zeek-side glue layer later.

removes comma and slight wording change. Also most places in the tutorial don't have a capital letter after the `:`

(the other place that a capital is after `:` is [here](https://github.com/zeek/spicy/blob/8d262f361a2810c26879104fbfebb61e107c82b9/doc/tutorial/index.rst?plain=1#L100))